pkg/webhook/policy.go

```diff
@@ -212,7 +221,7 @@ func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.
 		}
 	}
 	for _, r := range res {
-		if r.EnforcementAction == "deny" || r.EnforcementAction == "dryrun" {
+		if err := util.ValidateEnforcementAction(util.EnforcementAction(r.EnforcementAction)); err == nil {
```

Comment:
We should de-nest this code using continue:

```
if err := util.ValidateEnforcementAction(util.EnforcementAction(r.EnforcementAction)); err != nil {
   continue
}
```

Reply:
updated